cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/PostgreSqlStructuredTable.java

```diff
@@ -60,16 +65,20 @@
  */
 public class PostgreSqlStructuredTable implements StructuredTable {
   private static final Logger LOG = LoggerFactory.getLogger(PostgreSqlStructuredTable.class);
-  private static final int SCAN_FETCH_SIZE = 100;
+  private static final String SCAN_FETCH_SIZE = "data.storage.sql.scan.size.rows";
```

Comment:
nit: name it SCAN_FETCH_SIZE_ROWS, so it is clear what the unit of fetch size is

Reply:
Done